apps/emqx/test/emqx_mqtt_protocol_v5_SUITE.erl

```diff
@@ -209,6 +214,43 @@ t_connect_clean_start(Config) ->
 
     process_flag(trap_exit, false).
 
+-ifndef(BUILD_WITHOUT_QUIC).
+t_connect_clean_start_unresp_old_client(Config) ->
+    ConnFun = ?config(conn_fun, Config),
+    ClientID = atom_to_binary(?FUNCTION_NAME),
+    process_flag(trap_exit, true),
```

Comment:
Nit. Generally, not the best idea to trap exits here, as it sort of compromises CT's expectations about test code, and can lead to weird outcomes (at least in my "couple-of-years-ago" experience).

Reply:
L245 set it to `false`.

We need it because the emqtt client will crash in the middle of the test.

Reply:
I see.

> We need it because the emqtt client will crash in the middle of the test.

Yeah, I usually just unlink them if I don't care about their fate.

Reply:
Then we may end up leaving the zombie client process if clean up was forgotten. 

What is the downside of trapping exit in test?  My exp is traping EXIT let the TC fail with the amount of info needed,  rather than ct run crashed.

Reply:
Fair enough. That's a minor nit anyway.

> What is the downside of trapping exit in test?

I've seen it behave weirdly when coupled with testcase timeouts, i.e. CT runner gets confused by testcase process not dying when it's told to, logs strange error message and loses stacktrace information. This might have changed though, but I personally still prefer not to do that, basically in order not to deal with another fringe class of test failures.